### ISSUE TITLE
I'm in your repo.

### ISSUE THREAD

**Issue by harrisoninventiv** [rachel-kinkley/kassle#1]
Don't see code in here yet.  No rush.  Frequent small commits are better than worrying if a commit looks good or is at a good place to stop.

**Comment by harrisoninventiv** [comment]
Also, I primarily use Bitbucket, so I'm going to click all the buttons and see what I can do with my permissions.  Hope that is okay.

**Comment by rachel-kinkley** [comment]
Yes, I have yet to do any work beyond what you and I did yesterday. I'm waiting to get started until I turn in my last final papers, which should be around Monday. 

I think that is great advice around commits, thank you. 

Also, you are more than welcome to poke around however much you like. Thanks for the heads up!
